compose.yaml

```diff
@@ -1,6 +1,6 @@
 services:
   app:
-    image: bryanhonof/iot-api-server-app
+    image: monsterseppe/iot-api-server-app
```

Comment:
```suggestion
    image: bryanhonof/iot-api-server-app
```

Aangezien CI naar Dockerhub met mijn credentials gaat pushen, kunnen we deze het best laten als is.

Reply:
my bad die change moest ik nog reverten